vitrina/datasets/views.py

```diff
@@ -42,3 +42,20 @@ def get_context_data(self, **kwargs):
         }
         context_data.update(extra_context_data)
         return context_data
+
+
+class DatasetMembersView(ListView):
```

Comment:
This view should only be visible to users who are dataset's organization members (representatives).